app/api/ms_api_utils.py

```diff
@@ -0,0 +1,57 @@
+from flask import jsonify
+import requests
+# from requests.exceptions import HTTPError
+from flask import json
+
+from werkzeug.exceptions import HTTPException
+
+# set base url
+
+# for ms-api local server
+BASE_MS_API_URL = "http://127.0.0.1:4000"
+
+# for ms-api heroku server
+# BASE_MS_API_URL = "https://bridge-in-tech-ms-test.herokuapp.com"
+
+# @application.errorhandler(HTTPException)
+# def handle_exception(e):
+#     """Return JSON instead of HTML for HTTP errors."""
+#     # start with the correct headers and status code from the error
+#     response = e.get_response()
+#     # replace the body with JSON
+#     response.data = json.dumps({
+#         "code": e.code,
+#         "name": e.name,
+#         "description": e.description,
+#     })
+#     response.content_type = "application/json"
+#     return response
+
+# create instance
+def post_request(request_url, data):
+    response = None,
+    try:
+    
+        response_raw = requests.post(
+            request_url,
+            json = data,
+            headers = {"Accept": "application/json"}
+        )
+        response_raw.status_code = 201
+        response_raw.encoding = "utf-8"
+        response = response_raw.json()
+        
+    except HTTPException as e:
+        response = e.get_response()
+        response.data = json.dumps({
+            "code": e.code,
+            "name": e.name,
+            "description": e.description,
+        })
+        response.content_type = "application/json"
+
+    print(f"{response}")
```

Comment:
We should add proper logging rather than just printing

Reply:
yep, would work on that 👍

Reply:
@ramitsawhney27, do we want the logger to save to a file or print to console? Saving it to a file could become a problem coz it'll get bigger as the project grows.

Reply:
As this is only targetting MVP, can we leave the logger out for now? We can look at it again post GSoC. What do you thin, @ramitsawhney27 ?

Reply:
I think logging for the console should be something we should have for GSoC

Reply:
so, from what I can find, to print to console using `python logging`, is using `logging.warning`

<img width="943" alt="Screen Shot 2020-06-06 at 1 04 02 am" src="https://user-images.githubusercontent.com/29667122/83892200-a6472b80-a791-11ea-92ad-f5ff0da49a9f.png">

Is this ok?